recipe/meta.yaml

```diff
@@ -1,44 +1,38 @@
-{% set version = "4.1.5" %}
-{% set ver_maj_min_str = "-".join(version.split(".")[:2]) %}
+{% set name = "zeromq" %}
+{% set version = "4.2.1" %}
+{% set sha256 = "f68bc45b51297577522aa83d8f05727dba54e567a177330c82918454b656f74f" %}
 
 package:
-  name: zeromq
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: zeromq-{{ version }}.tar.gz
-  url: https://github.com/zeromq/zeromq{{ ver_maj_min_str }}/releases/download/v{{ version }}/zeromq-{{ version }}.tar.gz
-  sha1: 2b7490b77860be3060b1b1f92cd73184d309ca69
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/zeromq/libzmq/archive/v{{ version }}.tar.gz
```

Comment:
Yep, this happened to us quite a bit last summer. Also noticed this with a package just the other day. Got into a discussion about this with GitHub about it. They try to avoid checksum changes to autogenerated archives, but they make no guarantees. GitHub's recommendation is to use manually generated and uploaded archives if we want to ensure a static checksum.